web-client/src/components/RoleInfo/RoleInfo.tsx

```diff
@@ -0,0 +1,81 @@
+import { AlertTwoTone, HeartTwoTone } from '@ant-design/icons';
+import { Button, Col, Row } from 'antd';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+
+import cav from '../../assets/role_cav.png';
+import pin from '../../assets/role_pin.png';
+import TitleWithAddon from '../TitleWithAddon/TitleWithAddon';
+
+const StyledIntro = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 50px 50px;
+`;
+
+const Container = styled.div`
+  margin-top: 40px;
+`;
+
+const Box = styled.div`
+  background: #ffffff;
+  box-shadow: 0px 4px 12px rgba(0, 0, 0, 0.15);
+  border-radius: 4px;
+  margin: 10px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 310px;
+  height: 290px;
+`;
+
+const Pin = styled.img``;
+const Cav = styled.img``;
```

Comment:
These could probably be remove right? 😄

Reply:
I just left them because I was not sure how it behaves in termos of resizing. But I guess yes.